src/net_processing.cpp

```diff
@@ -2681,6 +2681,9 @@ class CompareInvMempoolOrder
     }
 };
 
+void MsgGetData(const Consensus::Params& consensusParams, CNode* pto, CConnman& connman, CNetMsgMaker& msgMaker, CNodeState& state, int64_t nNow, bool fFetch);
+void MsgFeeFilter(CNode* pto, CConnman& connman, CNetMsgMaker& msgMaker);
```

Comment:
I think CNetMsgMaker::Make could be const, which might be worth doing to be able to make this reference const.